ui/src/app/pages/workspace/workspace-edit-text.tsx

```diff
@@ -0,0 +1,409 @@
+    longDescription: <div>Primary focus of the research is drug/therapeutics development. The data
+      will be used to understand treatment-gene interactions or treatment outcomes relevant
+      to the therapeutic(s) of interest.</div>
+  },  {
+    shortName: 'commercialPurpose',
+    shortDescription: 'For-Profit Purpose',
+    longDescription: <div>The data will be used by a for-profit entity for research or product
+      or service development (e.g. for understanding drug responses as part of a
+      pharmaceutical company's drug development or market research efforts).</div>
+  }, {
+    shortName: 'educational',
+    shortDescription: 'Educational Purpose',
+    longDescription: <div>The data will be used for education purposes (e.g. for a college research
+      methods course, to educate students on population-based research approaches).</div>
+  }, {
+    shortName: 'otherPurpose',
+    shortDescription: 'Other Purpose',
+    longDescription: <div>If your Purpose of Use is different from the options listed above, please
+      select "Other Purpose" and provide details regarding your purpose of data use here
+      (500 character limit).</div>
+  }
+];
+ResearchPurposeItems.forEach(item => {
+  item.uniqueId = fp.uniqueId('research-purpose');
+});
+
+export const toolTipText = {
+  header: <div>A Workspace is your place to store and analyze data for a specific project.Each
+    Workspace is a separate Google bucket that serves as a dedicated space for file storage.
+    You can share this Workspace with other users, allowing them to view or edit your work. Your
+    Workspace is where you will go to build concept sets and cohorts and launch Notebooks for
+    performing analyses on your cohorts.</div>,
+  cdrSelect: <div>The curated data repository (CDR) is where research data from the <i>All of Us</i>
+    Research Program is stored. The CDR is periodically updated as new data becomes available for
+    use. You can select which version of the CDR you wish to query in this Workspace.</div>,
+  researchPurpose: <div>You  are required to describe your research purpose, or the reason why you
+    are conducting this study. This information, along with your name, will be posted on the
+    publicly available <i>All of Us</i> website (https://www.researchallofus.org/) to inform our
+    participants and other stakeholders about what kind of research their data is being used
+    for.</div>,
+};
+
+export const researchPurposeQuestions = [
+  {
+    header: ' What is the primary purpose of your project?',
+    description: <div>All fields required unless indicated as optional.</div>
+  }, {
+    header: <div> Please provide a summary of your research purpose by responding to the
+      questions below.</div>,
+    description: <div>Your responses should cover the major components of a research summary:
+      hypothesis, methods and anticipated findings. <strong>Your response will be displayed publicly
+        to
```

Comment:
Nit: The spaces feel super weird here. Can you clean it up? (There's only one word on a few lines)